dnscache.go

```diff
@@ -143,10 +144,7 @@ func (r *Resolver) Refresh() {
 	for _, addr := range addrs {
 		ctx, cancelF := context.WithTimeout(context.Background(), r.defaultLookupTimeout)
 		if _, err := r.LookupIP(ctx, addr); err != nil {
-			r.logger.Error("failed to refresh DNS cache",
-				zap.Error(err),
-				zap.String("addr", addr),
-			)
+			r.logger.Error("failed to refresh DNS cache", "addr", addr, "error", err)
```

Comment:
I think just removing logger is fine.
Because this error is meaningless until the cache is referenced, e.g., by Fetch, and if it is not cached, it calls lookup and returns the error to the user if it occurs.

Reply:
Thank you for your comment.

I replaced custom logger with classic log.Prinnf() that does not have any dependency. At the same time, it's easy to replaceable with zap.SetDefault and others methods alike in other loggers.

Would you want me to rollback other changes or is it okay to have slightly improved functionality and tests?